src/main/kotlin/com/github/adeynack/jsoncmp/ComparisonBase.kt

```diff
@@ -0,0 +1,25 @@
+package com.github.adeynack.jsoncmp
+
+import java.io.File
+
+abstract class ComparisonBase {
```

Comment:
That's just my preference but I strongly recommend interface over abstract class. 
Pros:
No state, do not have single inheritance restriction